pygbe/lspr.py

```diff
@@ -0,0 +1,321 @@
+"""
+This is the main function of the program.
+We use a boundary element method (BEM) to perform simple nanoparticle
+plasmonics. Localized surface plasmon resonance (LSPR) is an optical effect,
+but electrostatics is a good approximation in the long-wavelength limit. For
+nanoparticles smaller than the wavelength of incident light, PyGBe 
+compute the extinction cross-section.
+"""
+
+import os
+import re
+import sys
+import time
+import glob
+import numpy
+import pickle
+import subprocess
+from datetime import datetime
+from argparse import ArgumentParser
```

Comment:
We import the `read_inputs` from `pygbe.main` and we use that. I don't see why we should do a different one...